ci/gpu/build.sh

```diff
@@ -66,6 +66,7 @@ conda install -c nvidia -c rapidsai -c rapidsai-nightly -c conda-forge -c defaul
       distributed>=2.12.0 \
       dask-cudf=${MINOR_VERSION} \
       dask-cuda=${MINOR_VERSION} \
+      nccl>=2.5 \
```

Comment:
@rlratzel anything else we should do in `ci/` to enable OPG testing?

Reply:
I'm not very familiar with what an OPG build needs, but does MPI need to be enabled too?  If so, will you need to set the `BUILD_MPI` cmake flag?  If that's the case, you may also want to update the `build.sh` in the root of cugraph so users (and scripts like this one) can build from source correctly.

Reply:
MPI can be enabled as long as the machine have :
- MPI installed
-  multiple GPUs 
- call OPG gtests while spanning multiple processes

Considering these requirements this cannot be set as a default path for users in `build.sh` as of know.

My question was related to CI machines and capabilities to support this OPG testing from DevOps perspective though.

Reply:
> My question was related to CI machines and capabilities to support this OPG testing from DevOps perspective though.

@raydouglass do you know the answer to this question?

Reply:
Multi GPU testing is not available in gpuCI and I don't think MPI is installed. So sounds like you should not set OPG testing as default.